content/cloud-computing/concepts/paas/paas.md

```diff
@@ -0,0 +1,54 @@
+---
+Title: 'PaaS'
+Description: 'Platform as a service (PaaS): a service model that offers developers a platform to create, manage, and deploy their applications.'
+Subjects:
+  - 'Computer Science'
+  - 'Networking'
+Tags:
+  - 'Cloud'
+  - 'Platform'
+  - 'Development'
+  - 'Infrastructure'
+CatalogContent:
+  - 'foundations-of-cloud-computing'
+  - 'paths/back-end-engineer-career-path'
+---
+
+## Platform as a Service
+
+**Platform as a service (PaaS)** is a service model offered by cloud service providers that allows developers to develop and deploy apps without worrying about the underlying software and hardware their applications depend upon. When using a PaaS, the service provider provides and maintains the following:
+
+- Operating systems
+- Software development kits (SDK)
+- Frameworks
+- Libraries
+- APIs
+- Any other relevant software needed for application development
+
+In addition to the software, the service provider also provides the infrastructure needed for the application. PaaS includes the services provided by an infrastructure-as-a-service (IaaS) offering. Compared to the other two cloud services available - IaaS and software as a service (SaaS) - PaaS sits right in the middle, placing more responsibility on the service provider than IaaS and less responsibility than SaaS.
```

Comment:
Link to the SaaS and IaaS pages where appropriate.

Reply:
Not sure what "PaaS sits right in the middle, placing more responsibility on the service provider than IaaS and less responsibility than SaaS." means. It's not clear enough the amount of responsibility of the other two service types require.

Reply:
@HishamT, here are the entry links @nikstern is suggesting:

SaaS: `https://www.codecademy.com/resources/docs/cloud-computing/saas`
IaaS: `https://www.codecademy.com/resources/docs/cloud-computing/iaas`